README.md

```diff
@@ -11,4 +11,73 @@ However, if exceptions occur frequently, consider adjusting guidelines (by creat
 
 ## Table of contents
 
-// TODO
+- [Git](#git)
+  - [Workflow](#workflow)
+  - [Commit messages](#commit-messages)
+
+## [Git](#git)
+
+### [Workflow](#workflow)
+
+1. Create a feature branch  
+
+    - Prefix branch name with `feature/` or `fix/` when applicable; otherwise, prefix with your initials, e.g. `tk/`
+    - Use kebabcase
+    - Include the issue number when applicable
+    - Briefly describe the change  
+
+    Examples: `feature/610-seelog-wrapper`, `fix/tequilapi-startup`, `tk/polish`
```

Comment:
`tk/polish` is a bad example. Polish does not indicate anything that's meaningful and discernible, it's as good as `vk/breakfast`.

Reply:
If you're implementing breakfast, I see no problem with that. Small fixes should be encouraged, not discouraged, and if I can't afford half an hour deciding on the branch name, I'll just go with _polish_.